Ui/Component/Listing/Column/EQ8Score.php

```diff
@@ -17,29 +18,29 @@
 class EQ8Score extends Column
 {
     /**
-     * The order.
+     * The order helper (not, as you might assume, the order itself)
```

Comment:
Fairly high level of snark for a public repo. While I do see the value in pointing out the counterintuitiveness of Magento's class naming, it's possible that Adobe/Magento will read this while they're approving our code.

Reply:
it's actually referring to our codebase, but you're right about snark level - will refactor